apps/common/views/partials/add-another-address-summary.html

```diff
@@ -1,5 +1,5 @@
 {{#hasAddresses}}
-  <div id="storage-address-summary">
+  <div id="address-summary">
     <h2>Addresses added</h2>
```

Comment:
In case anyone picks up on this, this will be changed to a translation in a pr by me soon.